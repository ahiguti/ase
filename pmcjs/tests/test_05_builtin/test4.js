
var core = Perl.Functions['CORE'];
var pi = 3.14159265;

function dprint(x) {
  print(core.sprintf('%.4f', x));
}

dprint(core.abs(-17));
dprint(core.atan2(1, 5));
dprint(core.cos(1));
dprint(core.sin(1));
dprint(core.log(10));
dprint(core.exp(1));
dprint(core['int'](12.40));
dprint(core.sqrt(2));

core.srand(100);
dprint(core.rand(10000));

