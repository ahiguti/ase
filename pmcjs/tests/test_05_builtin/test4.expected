17.0000
0.1974
0.5403
0.8415
2.3026
2.7183
12.0000
1.4142
2510.5890
